Jenkinsfile.dev

```diff
@@ -112,12 +124,68 @@ if(hasRepoChanged){
               type: "button",
               text: "View Build Logs",
               style:"danger",        
-              url: "${currentBuild.absoluteUrl}/console"
+              APP_URL: "${currentBuild.absoluteAPP_URL}/console"
```

Comment:
Please fix all instances of `${currentBuild.absoluteAPP_URL}` -> `${currentBuild.absoluteUrl}`